gcj_mlc.c

```diff
@@ -121,6 +132,38 @@ GC_API void GC_CALL GC_init_gcj_malloc(int mp_index,
     UNLOCK();
 }
 
+/* Caller does not hold allocation lock. */
+GC_API void GC_CALL GC_init_gcj_vector (int mp_index,
```

Comment:
I don't think this is worth upstreaming, but in @ivmai in general should I put something custom like this in bdwgc itself? Or should I implement in my client code? If in bdwgc should I move to a separate file?